src/platforms/rp2/tests/test_erl_sources/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@
 # SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
 #
 
+include(ExternalProject)
```

Comment:
Just for my understanding: what is the purpose of this specific change?

Reply:
I belive it was required when I compiled this on some esoteric platform to try to reproduce CI issues.